charts/nri-bundle-legacy/.helmignore

```diff
@@ -14,7 +14,6 @@
 *.swp
 *.bak
 *.tmp
-*.orig
```

Comment:
git being :galaxymind

Reply:
Reminiscences of the `simple-nginx`... The legacy will perdure.